README.rst

```diff
@@ -51,6 +51,20 @@ the plugin to identify untested templates.
 If you get a :code:`django.core.exceptions.ImproperlyConfigured` error,
 you need to set the :code:`DJANGO_SETTINGS_MODULE` environment variable.
 
+If you get :code:`django_coverage_plugin.plugin.DjangoTemplatePluginException: Template debugging must be enabled in settings`, change your TEMPLATES setting:
```

Comment:
It seems that when using pytest, the check which can raise this exception doesn't actually run properly (#73) so it might help to mention here _"...or if you are using pytest and getting 0% coverage (issue #73)"_

Reply:
As of https://github.com/nedbat/coveragepy/commit/30c023b5b74f9c798645cbb3f35362ae046a4c25, the check is now a real Python warning, so pytest will display it.